src/drive/web/modules/views/Sharings/index.jsx

```diff
@@ -56,11 +55,11 @@ export const SharingsView = ({ router, sharedDocumentIds, children }) => {
     dispatch,
     router,
     location,
-    hasWriteAccess: false,
+    hasWriteAccess: true,
     canMove: true
   }
   const actions = useActions(
-    [share, download, trash, open, rename, move, qualify, versions, offline],
+    [download, trash, open, rename, move, qualify, versions, offline],
```

Comment:
same in the sharings view ? 🤔

Reply:
Same answer, it was never displayed there.